lib/changed_rpm_files_helper.rb

```diff
@@ -74,14 +74,14 @@ def parse_stat_line(line)
     user = uid if user == "UNKNOWN"
     group = gid if group == "UNKNOWN"
 
-    type = case(type)
-    when "directory"
-      "dir"
-    when "symbolic link"
-      "link"
-    when "regular file"
-      "file"
-    end
+    type = case type
```

Comment:
The whitespacing was correct before I think. I'll try to addapt hound accordingly, but please revert this change.